MediaPlayer/MediaControls.js

```diff
@@ -172,7 +178,16 @@ const MediaControls = kind({
 		 * @default 'shuffle'
 		 * @public
 		 */
-		shuffleIcon: PropTypes.string
+		shuffleIcon: PropTypes.string,
+
+		/**
+		 * Disables spotlight navigation into the component.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		spotlightDisabled: PropTypes.bool
```

Comment:
Do you leave the Could you please tell me why you made this add this prop in this ticket? (In PR description?)
If a prop is added, it may be necessary to modify the changelog and sampler.

Reply:
You are right, it wasn't in this ticket's scope. But since I added it, I will modify the changelog and sampler. Thank you!

Reply:
You are right, it wasn't in the ticket's scope. I will remove if for now and create a different ticket for that.

Reply:
I didn't mean to remove it, I just wanted to know a reason. 
Anyway, If necessary, it is ok to proceed with a separate ticket.
I welcome minor modifications for quality. We just need to leave a messege to keep history.